packages/server/src/fhir/routes.ts

```diff
@@ -23,8 +23,10 @@ import { sendOutcome } from './outcomes';
 import { Repository } from './repo';
 import { rewriteAttachments, RewriteMode } from './rewrite';
 import { smartConfigurationHandler, smartStylingHandler } from './smart';
+import { getConfig } from '../config';
 
 export const fhirRouter = Router();
+const router = new FhirRouter();
```

Comment:
If we move `options` to the constructor, then we can pass in `introspectionEnabled` here.

```ts
const router = new FhirRouter({ introspectionEnabled: getConfig().introspectionEnabled });
```